### PR TITLE
Add IndonesianMongabay classification dataset

### PR DESCRIPTION
**Pull request by ljvmiranda921**
New classification sentiment analysis dataset for indonesian

## Checklist for adding MMTEB dataset

Reason for dataset addition:

New dataset for indonesian language

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        type="Classification",
+        category="s2s",
+        date=None,
+        eval_splits=["validation", "test"],
+        eval_langs=["ind-Latn"],
+        main_score="accuracy",
+        form=["written"],
+        domains=["Web"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py:30]: Set license="Not specified"

File: mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        socioeconomic_status=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py:31]: socioeconomic_status="medium"

File: mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        annotations_creators=None,
+        dialect=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py:33]: dialect=[]

File: mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        name="IndonesianMongabayConservationClassification",
+        description="Conservation dataset that was collected from mongabay.co.id contains topic-classification task (multi-label format) and sentiment classification. This task only covers sentiment analysis (positive, neutral negative)",
+        reference="https://aclanthology.org/2023.sealp-1.4/",
+        dataset={
+            "path": "Datasaur/mongabay-experiment",
+            "revision": "c9e9f2c09836bfec57c543ab65983f3398e9657a",
+        },
+        type="Classification",
+        category="s2s",
+        date=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py:23]: Give the best estimate for the date.

File: mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        eval_splits=["validation", "test"],
+        eval_langs=["ind-Latn"],
+        main_score="accuracy",
+        form=["written"],
+        domains=["Web"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py:32]: annotations_creators="derived"

**Comment by ljvmiranda921** [comment]
Thanks for the reviews @shreeya-dhakal ! Updated :)

**Comment by shreeya-dhakal** [comment]
> Thanks for the reviews @shreeya-dhakal ! Updated :)

The update looks good to me. Your branch is out-of-date with main. Please merge main into this branch and re-run make lint and make test.

**Comment by ljvmiranda921** [comment]
I saw some test errors, will correct them in a bit

**Comment by ljvmiranda921** [comment]
Updated! Thanks again for the quick reviews @shreeya-dhakal 🙇

**Comment by shreeya-dhakal** [comment]
> > Thanks for the reviews @shreeya-dhakal ! Updated :)
> 
> The update looks good to me. Your branch is out-of-date with main. Please merge main into this branch and re-run make lint and make test.

@isaac-chung can you please look into this and approve this? Looks good to me.

File: results/intfloat__multilingual-e5-small/IndonesianMongabayConservationClassification.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{
+  "dataset_revision": "c9e9f2c09836bfec57c543ab65983f3398e9657a",
+  "mteb_dataset_name": "IndonesianMongabayConservationClassification",
+  "mteb_version": "1.7.58",
+  "test": {
+    "accuracy": 0.2632548618219038,
+    "accuracy_stderr": 0.028227484443608907,
+    "evaluation_time": 13.28,
+    "f1": 0.26091359988519114,
```
Comment (isaac-chung) [inline results/intfloat__multilingual-e5-small/IndonesianMongabayConservationClassification.json:9]: If there are only 3 classes, this seems to be below random chance. Wondering if you have more insights into that. (e.g. class imbalance).

File: docs/mmteb/points/634.jsonl [hunk 7]
```diff
@@ -0,0 +1 @@
+{"GitHub": "ljvmiranda921", "New dataset": 2}
```
Comment (isaac-chung) [inline docs/mmteb/points/634.jsonl:1]: Add review points.

**Review by isaac-chung — APPROVED** [review]
Thanks for adding this. Just a few small comments.

File: results/intfloat__multilingual-e5-small/IndonesianMongabayConservationClassification.json (hunk 6 above)
Comment (ljvmiranda921) [reply results/intfloat__multilingual-e5-small/IndonesianMongabayConservationClassification.json:9]: Hi @isaac-chung , yeah I think it's definitely class imbalance. When I checked there are more positive / neutral classes than that of negative ones.

**Comment by ljvmiranda921** [comment]
Updated! Thanks again for the reviews! @isaac-chung

File: mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+        dataset={
+            "path": "Datasaur/mongabay-experiment",
+            "revision": "c9e9f2c09836bfec57c543ab65983f3398e9657a",
+        },
+        type="Classification",
+        category="s2s",
+        date=("2012-01-01", "2023-12-31"),
+        eval_splits=["validation", "test"],
+        eval_langs=["ind-Latn"],
+        main_score="accuracy",
```
Comment (isaac-chung) [inline mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py:26]: Might be more appropriate for imbalanced datasets.
```suggestion
        main_score="f1",
```

**Comment by ljvmiranda921** [comment]
Hmm, not sure why the one test is failing 🤔

**Pull request merged**